src/assets/cache/parsers/cacheTextureArray.ts

```diff
@@ -5,8 +5,6 @@ import type { CacheParser } from '../CacheParser';
 
 /**
  * Returns an object of textures from an array of textures to be cached
- *
- * This will be added automatically if `pixi.js/assets` is imported
```

Comment:
if we remove all of these notes about importing `pixi.js/assets` we should still mention it somewhere that it may be required if you decide to manage your own imports